topo/dva/faucet_structural.yaml

```diff
@@ -0,0 +1,248 @@
+dps:
```

Comment:
Why is this faucet_structural.yaml and not just faucet.yaml?  I thought as per the design doc that faucet.yaml, in it's basic form, was the structural one?  Unless there's a clear reason for differentiation now (they're identical files, from what I can tell?) -- better to keep them the same and change them later.

Reply:
My intention was that faucet.yaml is used for minimal VLAN ACL test, while faucet_structural.yaml is used for test_topology as they are different test setups. Since they are the same, I am going to remove faucet_structural.yaml.

Reply:
Removed